### PR TITLE
Fix some bugs in the Log page

### PR DESCRIPTION
**Pull request by qiqizjl**
1: Fix When the log is not found, the prompt is English.
image

2:Fix When clicking the Refresh Log button is invalid. Set search box to null invalid bug

Signed-off-by: 耐小心 <qiqizjl@qq.com>

### CODE REVIEW

File: src/portal/lib/src/log/recent-log.component.ts [hunk 1]
```diff
@@ -67,9 +67,9 @@ export class RecentLogComponent implements OnInit {
     }
 
     public doFilter(terms: string): void {
-        if (!terms) {
-            return;
-        }
+        // if (!terms) {
```
Comment (jwangyangls) [inline src/portal/lib/src/log/recent-log.component.ts]: it has conflict because of I has modify it  in branch master ,please update code and commit .

Comment (jwangyangls) [reply src/portal/lib/src/log/recent-log.component.ts]: image

Comment (jwangyangls) [reply src/portal/lib/src/log/recent-log.component.ts]: Bona fide reminder: Your code should use the same commit (just like the pull request you are submitting now)

File: src/portal/lib/src/log/recent-log.component.html [hunk 2]
```diff
@@ -40,4 +40,4 @@
             </clr-dg-footer>
         </clr-datagrid>
     </div>
-</div>
```
Comment (zhoumeina) [inline src/portal/lib/src/log/recent-log.component.html]: please don't modify this line.

**Comment by jwangyangls** [comment]
I have resolved the conflict  in log.html and en.json,and approve it .

**Pull request merged**